cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -10,6 +10,9 @@
 {% if pillar['cluster_cidr'] is defined and pillar['cluster_cidr'] != "" -%}
   {% set cluster_cidr = "--cluster-cidr=" + pillar['cluster_cidr'] -%}
 {% endif -%}
+{% if pillar['service_cidr'] is defined and pillar['service_cidr'] != "" -%}
```

Comment:
It's actually `pillar['service_cluster_ip_range']` (I mentioned in https://github.com/kubernetes/kubernetes/pull/19242#issuecomment-218309287). I think that's probably why Jenkins is failing on you as well.